util/dvsim.py

```diff
@@ -362,6 +362,14 @@ def main():
         metavar="N",
         help="""Run only upto a fixed number of builds/tests at a time.""")
 
+    parser.add_argument(
+        "--local",
+        default=False,
+        action='store_true',
+        help=
+        """Deploy builds and runs on the local workstation instead of the compute farm."""
```

Comment:
perhaps mention this option isn't supported for now.

Reply:
Done, thanks.